snaps/concepts/apis.md

```diff
@@ -0,0 +1,226 @@
+---
+description: Learn about the Snaps APIs.
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# About the Snaps APIs
+
+Snaps, dapps, and MetaMask can communicate with each other using the [Snaps API](#snaps-api),
+[MetaMask JSON-RPC API](#metamask-json-rpc-api), and [custom JSON-RPC APIs](#custom-json-rpc-apis).
+
+## Snaps API
+
+Snaps can access the global object `snap`, which has one method: `request`.
+You can use this object to make [Snaps API](../reference/snaps-api.md) requests.
+These API methods allow Snaps to extend or modify the functionality of MetaMask.
+To call each method, you must first [request permission](../how-to/request-permissions.md) in the
+Snap manifest file.
+
+For example, to call [`snap_notify`](../reference/snaps-api.md#snap_notify):
+
+<Tabs>
+<TabItem value="Request permission">
```

Comment:
Are tabs good practice for presenting sequential information? I usually see it for presenting different ways to do the same thing, like a TypeScript and a JavaScript code sample in two tabs.

Reply:
The tabs were more to prevent cluttering this concepts page with code samples, but I went ahead and removed the tabs so the samples flow more naturally.